mw-webapp/src/component/label/Label.tsx

```diff
@@ -40,13 +39,15 @@ interface LabelProps {
 /**
  * Job tag component
  */
-export const LabelItem = observer((props: LabelProps) => {
+export const LabelItem = (props: LabelProps) => {
```

Comment:
Please change comment on 40 line Job tag component -> Label component
Also please check is it possible to rename component from Labelitem  to Label?